docs/runtime_suite/client-credentials/20_usage.md

```diff
@@ -446,7 +445,7 @@ Below are reported the authentication flows that are supported by Client Credent
 
 Below it is visible the sequence diagram of the login flow:
 
-<Mermaid chart={`
+```mermaid
```

Comment:
these changes must ported to the original repositories

Reply:
@MIA-Deltat1995 please remember to open a PR in the original repo or track it in an internal task